vercel.json

```diff
@@ -1438,6 +1438,62 @@
     {
       "source": "/docs/guides/migrate/deployment/deploy-database-changes-with-prisma-migrate",
       "destination": "/docs/guides/deployment/deploy-database-changes-with-prisma-migrate"
+    },
+    {
+      "source": "/docs/guides/deployment/deployment",
+      "destination": "/docs/guides/deployment/deploying-prisma"
+    },
+    {
+      "source": "/docs/guides/deployment/deployment-guides/deploying-to-heroku",
+      "destination": "/docs/guides/deployment/traditional-deployments/deploying-to-heroku"
+    },
+    {
+      "source": "/docs/guides/deployment/deployment-guides/deploying-to-koyeb",
+      "destination": "/docs/guides/deployment/traditional-deployments/deploying-to-koyeb"
+    },
+    {
+      "source": "/docs/guides/deployment/deployment-guides",
+      "destination": "/docs/guides/deployment/deploying-prisma"
+    },
+    {
+      "source": "/docs/guides/deployment/deployment-guides/deploying-to-azure-functions",
+      "destination": "/docs/guides/deployment/serverless-deployments/deploying-to-azure-functions"
+    },
+    {
+      "source": "/docs/guides/deployment/deployment-guides/deploying-to-vercel",
+      "destination": "/docs/guides/deployment/serverless-deployments/deploying-to-vercel"
+    },
+    {
+      "source": "/docs/guides/deployment/deployment-guides/deploying-to-aws-lambda",
+      "destination": "/docs/guides/deployment/serverless-deployments/deploying-to-aws-lambda"
+    },
+    {
+      "source": "/docs/guides/deployment/deployment-guides/deploying-to-netlify",
+      "destination": "/docs/guides/deployment/serverless-deployments/deploying-to-netlify"
+    },
+    {
+      "source": "/docs/guides/deployment/deployment-guides/deploying-to-deno-deploy",
+      "destination": "/docs/guides/deployment/edge-deployments/deploying-to-deno-deploy"
+    },
+    {
+      "source": "/docs/guides/deployment/deployment-guides/deploying-to-cloudflare-workers",
+      "destination": "/docs/guides/deployment/edge-deployments/deploying-to-cloudflare-workers"
+    },
+    {
+      "source": "/docs/guides/deployment/deployment-guides/deploying-migrations-from-a-local-environment",
+      "destination": "/docs/guides/deployment/deploying-migrations-from-a-local-environment"
+    },
+    {
+      "source": "/docs/guides/deployment/deployment-guides/caveats-when-deploying-to-aws-platforms",
+      "destination": "/docs/guides/deployment/caveats-when-deploying-to-aws-platforms"
+    },
+    {
+      "source": "/docs/guides/deployment/deployment-guides/deploying-to-a-different-os",
+      "destination": "/docs/guides/deployment/deploying-to-a-different-os"
+    },
+    {
+      "source": "/docs/guides/deployment/deployment-guides/use-prisma-with-serverless-framework",
+      "destination": "/docs/guides/deployment/use-prisma-with-serverless-framework"
```

Comment:
This will be integrated with the serverless folder, right?

Reply:
Yep! Anything AWS/Serverless Framework will be merged into the Serverless guide (in #5096 specifically)